src/Components/ShareIcon.css

```diff
@@ -0,0 +1,7 @@
+.shareIcon__icon {
```

Comment:
This CSS file is not needed if you just use primeflex inline classes for width, height, cursor , text color .

https://www.primefaces.org/primeflex/width

This is the official documentation on primeflex classes

Reply:
cool!